pymc_marketing/clv/models/gamma_gamma.py

```diff
@@ -287,6 +278,7 @@ def build_model(self):
             )
 
 
+# TODO: This model requires further evaluation and mention in a notebook.
```

Comment:
Can you please open an issue for this TODO please?

Reply:
Done 👍 For now I don't want to get carried away with refactoring this model, and prefer to keep things focused on the API changes.